samples/docker-domain/README.md

```diff
@@ -28,7 +28,9 @@ When the WDT `discoverDomain` tool is used on an existing domain, a ZIP archive
 
 ### How to Build and Run
 
-**NOTE:** The image is based on a WebLogic Server image in the docker-images project: `oracle/weblogic:12.2.1.3-developer`. Build that image to your local repository before building this sample.
+**NOTE:** The image is based on a WebLogic Server image in the Oracle Container Registry: `container-registry.oracle.com/middleware/weblogic:12.2.1.3-dev`. You will need to log in and accept the license agreement on the [Oracle Container Registry](https://container-registry.oracle.com/) before you can use Docker to pull the image.  
```

Comment:
I copied what you had before with the sample, just move the source location.  I thought you and Richard decided that you needed the Dev version because you needed the JDK and not just the server server JRE.

Reply:
verified JAR exists in that image.  Doc and Dockerfile changed.